hummingbot/connector/utils.py

```diff
@@ -6,6 +6,10 @@
     Optional
 )
 from zero_ex.order_utils import Order as ZeroExOrder
+from collections import namedtuple
+
+
+TradeFill_order_details = namedtuple("TradeFill_order_details", "market exchange_trade_id symbol")
```

Comment:
Might I ask what is this change for? It seems you have moved it from `markets_recorder.py`.

Reply:
This should not be merged, for whatever reason, hummingbot.py would always fail because it couldn't find `TradeFill_order_details`, this is an artefact of my debugging and should have been removed.

I reverted the change but am getting a similar error, even after I run: `./clean` and then `./compile`

Reply:
Issue resolved by running `./compile` twice.

Reply:
Reverting changes related to this